packages/auth/src/accessToken/service.ts

```diff
@@ -78,7 +78,7 @@ async function introspect(
   if (isTokenExpired(token)) {
     return undefined
   } else {
-    if (!token.grant || token.grant.state === GrantState.Revoked) {
+    if (!token.grant || isRevokedGrant(token.grant)) {
```

Comment:
At first I was thinking we should check for `isRejectedGrant` also, but I'm assuming we can't issue a token if a grant is finalized /rejected

Reply:
That's correct.